src/components/ChartScale/ChartScale.css

```diff
@@ -1,13 +1,13 @@
 .chart-scale {
     display: block;
-    position: absolute;
-    top: 0;
+    width: 100%;
 }
 
 .chart-scale__label-container {
     display: flex;
     border-top: 1px solid #ccc;
-    justify-content: space-evenly;
+    justify-content: space-between;
+    width: 100%;
 }
 
 .chart-scale__label {
```

Comment:
what do you think about adding `border-top: 1px solid black;` to chart-scale__label to give a sense of where the tick marks are on the y-axis? maybe it's not accurate at the top of the container... definitely not a big deal.

Reply:
actually, it seems like it would be pretty accurate, if we added that.